src/setup.js

```diff
@@ -1,40 +1,55 @@
 const mintAndDeposit = require('../tests/actions/mintAndDeposit');
 
-module.exports = async function(contracts, nodes, accounts, web3) {
+module.exports = async function(contracts, nodes, accounts, wallet) {
   const alice = accounts[0].addr;
 
-  let data = contracts.token.methods.addMinter(alice).encodeABI();
-  await contracts.governance.methods.propose(contracts.token.options.address, data).send({
-    from: alice, gas: 2000000
-  });
-  await contracts.governance.methods.finalize().send({ from: alice });
-  await contracts.token.methods.mint(alice, 500000000000).send({from: alice});
-  await contracts.token.methods.approve(contracts.exitHandler.options.address, 500000000000).send({from: alice});
-  await contracts.token.methods.approve(contracts.operator.options.address, 500000000000).send({from: alice});
+  let data = contracts.token.interface.functions.addMinter.encode([alice]);
+  let tx = await contracts.governance.propose(contracts.token.address, data, { gasLimit: 2000000 });
+  await tx.wait();
+  tx = await contracts.governance.finalize();
+  await tx.wait();
+  tx = await contracts.token.mint(alice, 500000000000);
+  await tx.wait();
+  tx = await contracts.token.approve(contracts.exitHandler.address, 500000000000);
+  await tx.wait();
+  tx = await contracts.token.approve(contracts.operator.address, 500000000000);
+  await tx.wait();
 
   for (let i = 0; i < nodes.length - 1; i++) {
-    const validatorInfo = await nodes[i].web3.getValidatorInfo();
-    const overloadedSlotId = `${contracts.operator.options.address}00000000000000000000000${i}`;
-    await contracts.governance.methods.setSlot(
-      overloadedSlotId, validatorInfo.ethAddress, `0x${validatorInfo.tendermintAddress}`
-    ).send({ from: alice, gas: 2000000 });
+    const validatorInfo = await nodes[i].getValidatorInfo();
+    const overloadedSlotId = `${contracts.operator.address}00000000000000000000000${i}`;
 
-    await web3.eth.sendTransaction({
-      from: alice,
+    tx = await contracts.governance.setSlot(
+      overloadedSlotId,
+      validatorInfo.ethAddress,
+      `0x${validatorInfo.tendermintAddress}`,
+      { gasLimit: 2000000 }
+    );
+    await tx.wait();
+
+    tx = await wallet.sendTransaction({
       to: validatorInfo.ethAddress,
-      value: web3.utils.toWei('1', "ether")
+      value: `0x${(10**18).toString(16)}`,
```

Comment:
could be also `utils.parseEther(1).toString(10)`

https://docs.ethers.io/ethers.js/html/api-utils.html#ether-strings-and-wei